sstables/mx/writer.cc

```diff
@@ -759,6 +767,32 @@ class writer : public sstable_writer::writer_impl {
         , _sst_schema(make_sstable_schema(s, _enc_stats, _cfg))
         , _run_identifier(cfg.run_identifier)
         , _write_regular_as_static(cfg.correctly_serialize_static_compact_in_mc && s.is_static_compact_table())
+        , _large_data_stats({{
+                {
+                    large_data_type::partition_size,
+                    large_data_stats_entry{
+                        .threshold = _sst.get_large_data_handler().get_partition_threshold_bytes(),
```

Comment:
with designated initializer, are the other fields guaranteed to be initialized or it's possibly UB?

Reply:
looks like this is defined behavior, all members are set to 0 if not explicitly initialized

Reply:
right. it's a valid concern but they are initialized to 0.

Reply:
thanks. it looks nice